indexing-service/src/main/java/io/druid/indexing/common/task/AppenderatorDriverRealtimeIndexTask.java

```diff
@@ -143,7 +143,7 @@ private static String makeTaskId(RealtimeAppenderatorIngestionSpec spec)
   private volatile FireDepartmentMetrics metrics = null;
 
   @JsonIgnore
-  private TaskMetricsGetter metricsGetter;
+  private RowIngestionMeters rowIngestionMeters;
```

Comment:
if this is injectable this could simply be `@JacksonInject`

Alternatively, it might make more sense to have a `Factory` object as the thing that is injected, and create the meters objects as needed in the classes.